opencv_flask/app.py

```diff
@@ -0,0 +1,32 @@
+from flask import Flask
+import cv2
+import numpy as np
+app = Flask(__name__)
+
+def display_vid(file_name):
```

Comment:
There's no need to comment inside the body of the method unless it's unclear by just reading it. Comment just on top of the method that explicitly tells us what this method do in a brief and concise manner.

Reply:
removed comment